studio/components/layouts/ProjectLayout/LayoutHeader/FeedbackDropdown/FeedbackDropdown.tsx

```diff
@@ -18,6 +18,8 @@ const FeedbackDropdown = () => {
     setIsOpen((isOpen) => !isOpen)
   }
 
+  let width = window.innerWidth
```

Comment:
I think there is a way that we can achieve this with tailwind right? Better if we stick to the same convention everywhere

Reply:
yup lets stick to tailwind classes for this: https://tailwindcss.com/docs/theme#screens

maybe something like
```
<span className="xs:hidden lg:block">Feedback on this page</span>
<span className="hidden lg:hidden">Feedback</span>
```

^ not the exact code - need to verify if they do work on the diff screen sizes